### ISSUE TITLE
When compiling in release mode in Xcode 16 Beta 6 , compilation fails

### ISSUE THREAD

**Issue by Gernot** [pointfreeco/swift-dependencies#264]
### Description

In 1.3.8, in WithDependencies.swift, compilation fails in Line 491:

```swift
  @_transparent
  private func isSetting<R>(
    _ value: Bool,
    isolation: isolated (any Actor)? = #isolation,
    operation: () async throws -> R
  ) async rethrows -> R {
    #if DEBUG
      try await DependencyValues.$isSetting.withValue(value, operation: operation)
    #else
      // ERROR:  Non-sendable type 'R' returned by implicitly asynchronous call to nonisolated function cannot cross actor boundary
      try await operation()
    #endif
  }
```

### Checklist

- [ ] I have determined whether this bug is also reproducible in a vanilla SwiftUI project.
- [ ] If possible, I've reproduced the issue using the `main` branch of this package.
- [X] This issue hasn't been addressed in an [existing GitHub issue](https://github.com/pointfreeco/swift-dependencies/issues) or [discussion](https://github.com/pointfreeco/swift-dependencies/discussions).

### Expected behavior

Project compiles

### Actual behavior

Project does not compile

### Dependencies version information

1.3.8

### Xcode version information

Xcode 16 Beta 6

**Comment by doxuto** [comment]
I'm facing this issue too, pls fix asap.

**Comment by mbrandonw** [comment]
This issue will be fixed with #265, which will be merged and released sometime today. In the meantime you can point to that branch directly if you can't wait.
